pkg/app/piped/platformprovider/terraform/terraform.go

```diff
@@ -313,7 +313,9 @@ func (t *Terraform) makeCommonCommandArgs() (args []string) {
 }
 
 var (
-	planHasChangeRegex = regexp.MustCompile(`(?m)^Plan: (\d+) to add, (\d+) to change, (\d+) to destroy.$`)
+	// Import block was introduced from Terraform v1.5.0.
+	// TODO: Show import in output.
+	planHasChangeRegex = regexp.MustCompile(`(?m)^Plan:(?: \d+ to import,)?? (\d+) to add, (\d+) to change, (\d+) to destroy.$`)
```

Comment:
Could you add a test to ensure this does not break piped with Terraform version before v1.5.0 🙏